zh/_author.md

```diff
@@ -1,10 +1,15 @@
 
 # 关于作者
 
-[Hugo Giraudel](http://hugogiraudel.com)，23 岁，来自法国的前端工程师，现居柏林，目前在 [Edenspiekermann](http://edenspiekermann.com) 工作。
+[Hugo Giraudel](http://hugogiraudel.com)，24 岁，来自法国的前端工程师，现居柏林，目前在 [Edenspiekermann](http://edenspiekermann.com) 工作。
 
 拥有多年的 Sass 开发经验，开发的项目包括：[SassDoc](http://sassdoc.com), [SitePoint Sass Reference](http://sitepoint.com/sass-reference/) 和 [Sass-Compatibility](http://sass-compatibility.github.io)。更多信息，请查阅这里的[项目列表](http://github.com/HugoGiraudel/awesome-sass)。
 
 已出版图书 [《CSS3 Pratique du Design Web》](http://css3-pratique.fr/)（法语版）和[《Jump Start Sass》](https://learnable.com/books/jump-start-sass)（英语版）。
 
-{% include partials/twitter.html %}
+<div class="button-wrapper">
```

Comment:
It should be `{% include partials/twitter.html %}` here. I can still fix it myself though.